.zenodo.json

```diff
@@ -1,13 +1,13 @@
 {
   "upload_type": "dataset",
-  "description": "<p>List, Johann Mattis &amp; Tjuka, Annika &amp; Rzymski, Christoph &amp; Greenhill, Simon &amp; Schweikhard, Nathanael &amp; Forkel, Robert (eds.) 2022. Concepticon vX.Y.Z. A Resource for the Linking of Concept Lists. Leipzig: Max Planck Institute for Evolutionary Anthropology. Available online at https://concepticon.clld.org</p>",
+  "description": "<p>List, Johann Mattis &amp; Tjuka, Annika &amp; Rzymski, Christoph &amp; Greenhill, Simon &amp; Forkel, Robert (eds.) 2022. Concepticon v3.0.0. A Resource for the Linking of Concept Lists. Leipzig: Max Planck Institute for Evolutionary Anthropology. Available online at https://concepticon.clld.org</p>",
   "alternate_identifiers": [
     {
       "scheme": "url",
       "identifier": "https://concepticon.clld.org"
     }
   ],
-  "title": "CLLD Concepticon X.Y.Z as CLDF dataset",
+  "title": "CLLD Concepticon 3.0.0 as CLDF dataset",
```

Comment:
@LinguList @AnnikaTjuka are you ok with this as title for the dataset?

Reply:
Yes, that looks good to me!

Reply:
For me too.